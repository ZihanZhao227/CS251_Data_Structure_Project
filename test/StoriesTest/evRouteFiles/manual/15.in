3
0 0 0 CITY Start
1 10000 0 CHARGING_STATION CS1
2 20000 0 CITY End
