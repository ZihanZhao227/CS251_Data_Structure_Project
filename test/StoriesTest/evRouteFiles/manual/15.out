0 2 15000.000 20000.000
0 1 2
